src/evo/cbtx.cpp

```diff
@@ -30,10 +32,16 @@ bool CheckCbTx(const CTransaction& tx, const CBlockIndex* pindexPrev, TxValidati
         return state.Invalid(TxValidationResult::TX_CONSENSUS, "bad-cbtx-payload");
     }
 
-    if (cbTx.nVersion == 0 || cbTx.nVersion > CCbTx::CURRENT_VERSION) {
+    if (cbTx.nVersion == 0 || cbTx.nVersion > CCbTx::CB_V20_VERSION) {
```

Comment:
shoudn't change this? should bump CURRENT_VERSION no?

Reply:
`CURRENT_VERSION` was renamed to `CB_V19_VERSION`.